packages/patternfly-4/react-table/src/components/Table/index.d.ts

```diff
@@ -12,7 +12,8 @@ export {
   IColumn,
   IExtra,
   IExtraData,
-  IExtraColumnData
+  IExtraColumnData,
+  ITransform
```

Comment:
sorry, i think i am missing something here. Tested these `.d.ts` downstream... getting 

```
Module '"../../../../../../../../../../Users/priley/GitHub/patternfly-react/node_modules/@patternfly/react-table/src/components/Table/Table"' has no exported member 'ITransform'.ts(2305)
```

Reply:
changes in `Table.d.ts` work fine for me though.

Reply:
Yeah it wasn't renamed from `ITransform` to `IDecorator`. Good catch!

Reply:
🤦‍♂️